### PR TITLE
solution

### PR DESCRIPTION
**Pull request by RomSmile**

### CODE REVIEW

**Review by evgen1sider — APPROVED** [review]
good job

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+    super(name, hidden);
+    this.hidden = false;
+  }
+
+  hide() {
+    if (!this.hidden) {
+      this.hidden = true;
+    } else {
+      this.hidden = false;
+    }
```
Comment (VinnieJ-2k20) [inline src/herbivoresAndCarnivores.js:25]: Let's simplify this with a toggler function.
```suggestion
    this.hidden = !this.hidden;
```

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
 class Carnivore extends Animal {
-  // write your code here
+  // eslint-disable-next-line no-useless-constructor
+  constructor(name, health) {
+    super(name, health);
+  }
+
+  bite(animal) {
+    if (animal instanceof Herbivore) {
+      if (!animal.hidden) {
```
Comment (VinnieJ-2k20) [inline src/herbivoresAndCarnivores.js:37]: Don't write nested `if`s, use logical operators.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Almost done, let's improve your code.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**